pkg/recommender.py

```diff
@@ -694,6 +657,9 @@ def _schedule_evt(self):
                 #send the blank message after everything for both morning and evening messages-------------
                 _ = call_ema('1', '995', alarm='false', test=self.test_mode)
```

Comment:
@jasonjabbour I guess this is the place that crashed the scheduling event thread.
coz we lost the EMA, keep calling it outside of `try..catch..` will throw new exceptions without being catched. 
we should move it within `try`

Reply:
Done!